### PR TITLE
Fix tiktok.com, buffer.com vectors and add tracker.gg to dark-sites

### PR DESCRIPTION
**Pull request by zayadur**
Add INVERT on svg.StyledLinkLogo and svg.SvgPlayIcon on TikTok webapp.
Add tracker.gg to collection of already dark sites.

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -2456,6 +2456,13 @@ CSS
 
 ================================
 
+buffer.com
+
+INVERT
+a[class*="BufferLogoWithWords"]
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:2462]:
```suggestion
IGNORE INLINE STYLE
div[class^="style__LogoWrapper"] *
```

**Review by Gusted — APPROVED** [review]

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (zayadur) [reply src/config/dynamic-theme-fixes.config]: This was a learning experience. Thank you for the review!

File: src/config/dark-sites.config [hunk 2]
```diff
@@ -949,6 +949,7 @@ title-case-converter.vercel.app
 tmtheme-editor.herokuapp.com
 toneden.io
 totalwarwarhammer.gamepedia.com
+tracker.gg
```
Comment (Gusted) [inline src/config/dark-sites.config:952]:
```diff
    -   "tracker.gg",
        "tracker.fumik0.com",
    +   "tracker.gg",
```

Please re-order it.

Comment (zayadur) [reply src/config/dark-sites.config]: Whoopsie. Fixed.

**Pull request merged**